pureStorageDriver/values.schema.json

```diff
@@ -0,0 +1,1742 @@
+                                            },
+                                            "topology.purestorage.com/region": {
+                                                "$id": "#/properties/arrays/properties/FlashBlades/items/anyOf/0/properties/Labels/properties/topology.purestorage.com/region",
+                                                "type": "string",
+                                                "title": "The topology.purestorage.com/region schema",
+                                                "default": "",
+                                                "examples": [
+                                                    "region-0"
+                                                ]
+                                            },
+                                            "topology.purestorage.com/zone": {
+                                                "$id": "#/properties/arrays/properties/FlashBlades/items/anyOf/0/properties/Labels/properties/topology.purestorage.com/zone",
+                                                "type": "string",
+                                                "title": "The topology.purestorage.com/zone schema",
+                                                "default": "",
+                                                "examples": [
+                                                    "zone-0"
+                                                ]
+                                            }
+                                        }
+                                    },
+                                    "MgmtEndPoint": {
+                                        "$id": "#/properties/arrays/properties/FlashBlades/items/anyOf/0/properties/MgmtEndPoint",
+                                        "type": "string",
+                                        "title": "The MgmtEndPoint schema",
+                                        "default": "",
+                                        "examples": [
+                                            "1.2.3.5"
+                                        ]
+                                    },
+                                    "NFSEndPoint": {
+                                        "$id": "#/properties/arrays/properties/FlashBlades/items/anyOf/0/properties/NFSEndPoint",
+                                        "type": "string",
+                                        "title": "The NFSEndPoint schema",
+                                        "default": "",
+                                        "examples": [
+                                            "1.2.3.5"
+                                        ]
+                                    }
+                                }
+                            }
+                        ],
+                        "$id": "#/properties/arrays/properties/FlashBlades/items"
+                    }
+                }
+            }
+        },
+        "clusterID": {
```

Comment:
is there a way to validate that customer is not using the default value `REPLACE_CLUSTER_ID_WITH_A_UNIQUE_VALUE_FOR_YOUR_CLUSTER `? and is it possible to add a length constraint to this thing?
cc: @dsupure do you know what is the constraint for clusterID length?

Reply:
there should be ways to do both, just let me know the exact requirement.

Reply:
and good idea